docs/docs/how-tos/memory.ipynb

```diff
@@ -0,0 +1,1089 @@
+{
+ "cells": [
+  {
+   "attachments": {},
+   "cell_type": "markdown",
+   "id": "15ed7413-c876-4d38-b080-79c71133549b",
+   "metadata": {},
+   "source": [
+    "# Manage Conversation History\n",
```

Comment:
Maybe "Optimize memory" instead?

We also probably want to cross link with the Persistence how-tos in the intro here too.